app/src/main/java/org/schabi/newpipe/settings/custom/NotificationActionsPreference.java

```diff
@@ -1,10 +1,10 @@
-package org.schabi.newpipe.settings;
+package org.schabi.newpipe.settings.custom;
```

Comment:
Why change package?

Reply:
There is `DurationListPreference` in that package which is a custom preference.
Since this is also a custom preference i put it in there.

Reply:
Ok, good